HomeConMenu/Resources/ja.lproj/Acknowledgments.html

```diff
@@ -16,7 +16,7 @@ <h1>謝辞</h1>
 <section class="CotEditor">
 <h3>CotEditor</h3>
 <p><a href="https://github.com/coteditor/CotEditor">https://github.com/coteditor/CotEditor</a></p>
-<p>環境設定のビューは，CotEditorのコードを参考にしています．</p>
+<p>設定のビューは，CotEditorのコードを参考にしています．</p>
```

Comment:
The punctuation in this line differs from the rest.
（、。 => ，．）

Reply:
Nice. I've just added a new commit: https://github.com/sonsongithub/HomeConMenu/pull/7/commits/561eacd70ea372975be07253da2e13ce1e475231